src/hooks/project/useProject.tsx

```diff
@@ -42,6 +44,22 @@ const useProject = () => {
     setLoading(false)
   }
 
+  const setSplit = (split: any) => {
+    const temp = [...splits]
+    temp.push(split)
+    setSplits([...temp])
+  }
+
+  const removeSplit = (index: number) => {
+    let temp = [...splits]
+    temp.splice(index, 1)
+    setSplits([...temp])
+
+    temp = [...splitPercents]
+    temp.splice(index, 1)
+    setSplitPercents([...temp])
+  }
```

Comment:
optional: splits should be managed in a `useSplits` hook.
- useProject can still import the hook, just keeps new logic modular